src/Components/AddReservation.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { addReservation } from '../Redux/reservations/addReservationsSlice';
+import '../AddReservation.css';
+
+function AddReservation() {
```

Comment:
- Please, consider using arrow funcs according to Javascript ES6 good practices.